addons/udes_stock/models/stock_picking.py

```diff
@@ -930,3 +930,18 @@ def open_back_orders(self):
         return self.open_related_pickings(
             self.u_created_backorder_ids.ids, "Created Back Orders"
         )
+
+    @staticmethod
+    def _get_package_search_domain(package):
+        """
+        Generate the domain for searching pickings that use a package.
+        Useful as this can be overridden in other modules if functionality is expanded.
+
+        :args:
+            - package: a stock.quant.package object
+        """
+        return [
+            "|",
+            ("move_line_ids.package_id", "=", package.id),
+            ("move_line_ids.result_package_id", "=", package.id),
```

Comment:
Question: Does result_package_id exist in udes14?

Reply:
On stock move line model, yes. Look at Odoo stock module,  stock.move model.